test/util/mock-helpers.js

```diff
@@ -0,0 +1,16 @@
+const request = require('request');
+const E2eHelpers = require('./e2e-helpers');
+
+function mock(token, json) {
+  const jsonWithToken = token ? {
+    ...json, auth: `Token token=${token}`
+  } : json;
+
+  request({
+    uri: `${E2eHelpers.apiUrl}/mock`,
+    method: 'POST',
+    json: jsonWithToken
+  });
+}
+
+module.exports = mock;
```

Comment:
This is somewhat out of convention - might consider changing this to export multiple methods or singularizing the name of the file.